### DISCUSSION TITLE
Are you going to continue this game?

### DISCUSSION THREAD

**Discussion by farisfaikar-r** [farisfaikar/Number-Game#24] [Q&A] [answered]
This is a pretty neat simple game but as of currently it looks unfinished. Looks like the achievement system isn't working yet and the file structure for this project is outdated. Is there any plan to make it an executable as well?

**Comment by farisfaikar** [comment, marked as answer]
Hello me. Yes, some features don't work such as the achievement system because it was only there as a proof of concept. I do not intend to continue this project as of now. Maybe in the future when the stars are aligned or if there's a particular reason for me to continue or update this project, I might add the achievement system correctly and also make an executable for it as well. Cheers
